docs/docs/widgets/tree-select.md

```diff
@@ -5,9 +5,9 @@ title: Tree Select
 
 # Tree Select
 
-The **Tree Select** widget is a group checkboxes in a TreeView which can be expanded or collapsed.
+The **Tree Select** component is a group checkboxes in a TreeView which can be expanded or collapsed.
 
-<div style={{paddingTop:'24px', paddingBottom:'24px'}}>
+<div style={{padding-top:'24px'}}>
```

Comment:
Hey @hahaanisha, you have to use `paddingTop` only and not `padding-top`, please update this wherever necessary.